src/components/SortOption/SortOption.tsx

```diff
@@ -0,0 +1,74 @@
+import classNames from 'classnames';
+import { useLocation } from 'react-router-dom';
+import { SearchLink } from '../SearchLink/SearchLink';
+
+type Props = {
+  option: string;
+};
+
+export const SortOption: React.FC<Props> = ({ option }) => {
+  const location = useLocation();
+
+  return (
+    <th>
+      <span
+        className="is-flex is-flex-wrap-nowrap"
+      >
+        {option}
+        {location.search.includes('&order=desc')
+          ? (
+            <SearchLink
+              params={
+                location.search.includes(`sort=${option.toLowerCase()}`)
+                  ? {
+                    sort: null,
+                    order: null,
+                  }
+                  : {
+                    sort: option.toLowerCase(),
+                    order: null,
+                  }
```

Comment:
better to prepare values before return - and then use variables here
It will increase readability